### PR TITLE
chore: allow slack/bolt 3.x

### PR DESCRIPTION
**Pull request by mattcasey**
**Related Issue**  
Supports #99 

**Related PRs**  
This PR is not dependent on any other PR

**What does this PR do?**  
Removes a warning about peer dependencies for @slack/bolt v3

**Description of Changes**  
Allows using this library with bolt v3

**What gif most accurately describes how I feel towards this PR?**  
Example of a gif

### CODE REVIEW

**Comment by mattcasey** [comment]
Not sure if I should be bumping the version of koa-bolt in this PR or not, let me know, thanks!

**Comment by barlock** [comment]
> Not sure if I should be bumping the version of koa-bolt in this PR

Please do, I'd like to make sure that the test cases for `koa-bolt` pass with this new version. It appears compatible, but I want to double check. 

I took a quick pass locally and it looks like it should be fine for `koa-bolt` but some of the other packages have breaking type changes at least. If you could bump it for just `koa-bolt` I'll take a pass after to get the rest of the packages upgraded. 

🙇 Thanks for the PR.

**Comment by mattcasey** [comment]
OK, bumped the version to 1.2.1. Thanks for looking at the other packages

File: packages/koa-bolt/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack-wrench/koa-bolt",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (barlock) [inline packages/koa-bolt/package.json:3]:
```suggestion
  "version": "1.2.0",
```

😅 Sorry, I meant update the [`devDependancy` of `@slack/bolt`](https://github.com/IBM/slack-wrench/pull/101/files#diff-fcf87f9b4ff934fbb9a332b6395310282b96d926e8182986f2c31c374c9ba40fR28) (line 28) so the test cases run against the newer version.

I'll handle bumping this and rolling a release for you.

**Review by barlock — CHANGES_REQUESTED** [review]

File: packages/koa-bolt/package.json (hunk 1 above)
Comment (mattcasey) [reply packages/koa-bolt/package.json]: Woops, that makes a lot more sense!

**Review by barlock — APPROVED** [review]
:shipit: LGTM, thanks for the PR.

**Comment by barlock** [comment]
[1.3.0](https://www.npmjs.com/package/@slack-wrench/koa-bolt) is published 🚀

**Pull request merged**